.coderabbit.yaml

```
@@ -0,0 +1,35 @@
+# Configuration for development teams
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)